tests/test-awful-client.lua

```diff
@@ -264,6 +264,12 @@ table.insert(multi_screen_steps, function()
                 screen = screen[1],
             })
 
+    -- Same as previous, but switched
```

Comment:
For the commit message: How does this fix any issue? Should that be in the previous commit's commit message instead?